blocks/agent-address/agent-address.js

```diff
@@ -0,0 +1,22 @@
+import { getMetadata } from '../../scripts/aem.js';
+import {
+  a, div, p,
+} from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  const streetAddress = getMetadata('streetaddress');
+  const addressLocality = getMetadata('addresslocality');
+  const addressRegion = getMetadata('addressregion');
+  const postalCode = getMetadata('postalcode');
+
+  const textDiv = div({ class: 'address' },
+    p({}, 'Berkshire Hathaway HomeServices'),
```

Comment:
```suggestion
    p('Berkshire Hathaway HomeServices'),
```
If no attributes for the tag are needed, you don't have to provide the object.